pkg/logutils/logutils.go

```diff
@@ -161,14 +161,12 @@ func ConfigureLogging(configParams *config.Config) {
 		log.WithError(fileOpenErr).WithField("file", configParams.LogFilePath).
 			Fatal("Failed to open log file.")
 	}
-	if sysErr != nil {
```

Comment:
What's the reasoning behind removing this check? I think it looks like a valid check.

Reply:
One of the linters says sysErr can never be nil and from reading the source that seems correct, but I'm open to suggestions.

Reply:
I think it would be nil if `configParams.LogSeveritySys` is `""` since nil is the default value for error. So I'm not sure how how it could never be nil.
Right? or is there something I'm missing.